docs/website/blog/2023-12-01-dlt-kestra-demo.md

```diff
@@ -0,0 +1,170 @@
+:::
+
+## The data loading part
+
+> 💡 This is entirely managed by `dlt` in just five lines of code.
+
+I set up a pipeline using the **[Inbox](https://dlthub.com/docs/dlt-ecosystem/verified-sources/inbox)** source – a regularly tested and verified source from **`dlt`** – with BigQuery as the destination.
+
+In my scenario, the email data doesn't have nested structures, so there's no need for flattening. However, if you encounter nested structures in a different use case, **`dlt`** can automatically normalize them during loading.
+
+Here's how the pipeline is defined and subsequently run in the first task of the main flow in **`Kestra`**:
+
+```python
+# Run dlt pipeline to load email data from gmail to BigQuery
+      pipeline = dlt.pipeline(
+          pipeline_name="standard_inbox",
+          destination='bigquery',
+          dataset_name="messages_data",
+          full_refresh=False,
+      )
+
+      # Set table name
+      table_name = "my_inbox"
+      # Get messages resource from the source
+      messages = inbox_source(start_date = pendulum.datetime(2023, 11, 15)).messages
+      # Configure the messages resource to get bodies of the emails
+      messages = messages(include_body=True).with_name(table_name)
+      # Load data to the "my_inbox" table
+      load_info = pipeline.run(messages)
+```
+
+In this setup ☝️, **`dlt`** loads all email data into the table “my_inbox”, with the email body specifically stored in the “body” column. After executing your flow in **`Kestra`**, the table in BigQuery should appear as shown below:
+
+![bigquery_my_inbox](https://storage.googleapis.com/dlt-blog-images/dlt_kestra_bigquery_my_inbox.png)
+
+:::tip
+This implementation doesn't handle email attachments, but if you need to analyze, for instance, invoice PDFs from your inbox, you can read about how to automate this with `dlt` [here](https://dlthub.com/docs/blog/dlt-ops-startups).
+:::
+
+## The AI part
+
+>💡 In this day and age, how can we not incorporate AI into everything? 😆
+
+But seriously, if you're familiar with OpenAI, it's a matter of an API call to the chat completion endpoint. What simplifies it even further is `Kestra`’s [OpenAI plugin](https://kestra.io/plugins/plugin-openai).
+
+In my [subflow](https://github.com/dlt-hub/dlt-kestra-demo/blob/main/subflow.yml), I used it to obtain both the summary and sentiment analysis of each email body. Here's a glimpse of how it's implemented:
+
+```yaml
+- id: get_summary
+  type: io.kestra.plugin.openai.ChatCompletion
+  apiKey: "{{ secret('OPENAI_API') }}"
+  model: gpt-3.5-turbo
+  prompt: "Summarize the email content in one sentence with less than 30 words: {{inputs.data[0]['body']}}"
+  messages: [{"role": "system", "content": "You are a tool that summarizes emails."}]
+```
+
+:::info
+`Kestra` also includes Slack, as well as BigQuery plugins, which I used in my flows. Additionally, there is a wide variety of [other plugins](https://kestra.io/plugins) available.
```

Comment:
you often use bold font for the words **`Kestra`** and **`dlt`** (which looks like ChatGPT generated, and that's fine), but my point is that you should use the same style for them , make them all bold or make them all non-bold.